src/sendbuf.c

```diff
@@ -234,13 +235,61 @@ void ExpandMacro(void) {
 	replace_all(buffer, BUFSIZE, "|", "");	    /* drop it */
 }
 
+void ExpandMacro_PreviousQso(void) {
+    int i;
+    static char qsonroutput[5] = "";
+    static char rst_out[4] = "";
+
+    struct qso_t *prev_qso = g_ptr_array_index(qso_array, NR_QSOS - 1);
```

Comment:
what if there's no previous QSO?

in a networked setup our previous QSO is not necessarily the one before the last on the list.

Reply:
What do you recommend to look at in the case of networked setup?

Reply:
Maybe I'm missing something, but lan seems to be broken in master. When I enter a qso in one node the other node is seg faulting.

Reply:
Yes, network is broken now and needs a rework. So let's for the moment just add a function like `get_previous_qso()` which simply does the same as line 243. Later we can then update it to match the network code.

Probably `qso_t` will have to be extended with a `node` field to allow selection of own QSOs.